### DISCUSSION TITLE
Welcome to Animata Discussions!

### DISCUSSION THREAD

**Discussion by hari** [codse/animata#366] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by webdevNYC** [comment]
Hi everyone! 
I’m Adriana, a front-end developer passionate about building clean, interactive, and user-friendly interfaces. My background combines design and code. I work with React, Tailwind, and modern UI frameworks, but I also bring my eye for motion and detail from a strong design and UX.
Quick question, will Animata join Hacktoberfest again this year like last time? Would love to contribute!
